Dockerfile.builder

```diff
@@ -27,11 +27,11 @@ RUN test -n $USE_GO_VERSION_FROM_WEBSITE \
     && rm -f go1.10.linux-amd64.tar.gz
 ENV PATH=$PATH:/usr/local/go/bin
 
-# Get glide for Go package management
-RUN cd /tmp \
-    && wget --no-verbose https://github.com/Masterminds/glide/releases/download/v0.13.1/glide-v0.13.1-linux-amd64.tar.gz \
-    && tar xzf glide-v*.tar.gz \
-    && mv linux-amd64/glide /usr/bin \
-    && rm -rfv glide-v* linux-amd64
-
+# Get dep for Go package management and make sure the directory has full rwz permissions for non-root users
+RUN mkdir -p /tmp/go/bin && chmod a+rwx /tmp/go
+ENV GOPATH /tmp/go
```

Comment:
I would switch the order of these two lines and use the env var:

```Dockerfile
ENV GOPATH /tmp/go
RUN mkdir -p $GOPATH/bin && chmod a+rwx $GOPATH
```

Reply:
ok, let me do that